### PR TITLE
Add GeneratedCode attribute

### PR DESCRIPTION
**Pull request by skarllot**
## The issue or feature being addressed
- #150

## Details on the issue fix or feature implementation
- Add `GeneratedCode` attribute to every generated class, including attributes

### CODE REVIEW

File: src/Jab/ContainerGenerator.cs [hunk 1]
```diff
@@ -276,6 +280,7 @@ private void Execute(GeneratorContext context)
                             codeWriter.Line($";");
                         }
 
+                        codeWriter.LineRaw(_generatedCodeAttribute);
```
Comment (pakrym) [inline src/Jab/ContainerGenerator.cs:283]: Do you need the attribute for nested classes?

File: src/Jab/ContainerGenerator.cs [hunk 2]
```diff
@@ -595,9 +600,12 @@ public override void Initialize(AnalysisContext context)
 
     private static string ReadAttributesFile()
     {
+        const string placeholder = "#if JAB_ATTRIBUTES_PACKAGE";
```
Comment (pakrym) [inline src/Jab/ContainerGenerator.cs:603]: Do you mind adding the attribute directly in the Attributes.cs file?

Comment (skarllot) [reply src/Jab/ContainerGenerator.cs:603]: What about the version number?

File: src/Jab/ContainerGenerator.cs (hunk 1 above)
Comment (skarllot) [reply src/Jab/ContainerGenerator.cs:283]: I think it's not necessary

File: src/Jab/ContainerGenerator.cs (hunk 2 above)
Comment (pakrym) [reply src/Jab/ContainerGenerator.cs:603]: [It looks like](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/CodeDom/Compiler/GeneratedCodeAttribute.cs#L15) the version parameter is nullable. I'd be fine with passing `null`.

File: src/Jab/ContainerGenerator.cs (hunk 1 above)
Comment (skarllot) [reply src/Jab/ContainerGenerator.cs]: Removed

File: src/Jab/ContainerGenerator.cs (hunk 2 above)
Comment (skarllot) [reply src/Jab/ContainerGenerator.cs]: Done

**Review by pakrym — APPROVED** [review]

**Comment by pakrym** [comment]
Thank you @skarllot !

**Pull request merged**